string motion_type
---
bool response
